numpyro/contrib/autoguide/__init__.py

```diff
@@ -326,11 +326,11 @@ class AutoIAFNormal(AutoContinuous):
     :param `**arn_kwargs`: keywords for constructing autoregressive neural networks, which includes:
 
         * **hidden_dims** (``list[int]``) - the dimensionality of the hidden units per layer.
-            Defaults to ``[latent_size, latent_size]``.
+          Defaults to ``[latent_size, latent_size]``.
```

Comment:
4 indent here leads to [unintended format](https://numpyro.readthedocs.io/en/latest/autoguide.html#autoiafnormal)